### ISSUE TITLE
Add wreck to performance comparisons

### ISSUE THREAD

**Issue by lance** [bucharest-gold/roi#19]
https://www.npmjs.com/package/wreck

**Comment by helio-frota** [comment]
Also add node-fetch
https://www.npmjs.com/package/node-fetch

**Comment by helio-frota** [comment]
first time:

```
Scores: (bigger is better)

roiGET
Raw:
 > 6.983016983016983
 > 6.905094905094905
 > 7.246753246753247
 > 6.383616383616384
 > 6.931750741839763
 > 6.05994005994006
 > 7.004955401387512
 > 7.060939060939061
Average (mean) 6.822008347823489

wreckGET
Raw:
 > 6.863136863136863
 > 6.35964035964036
 > 6.5994005994005995
 > 6.923076923076923
 > 6.20979020979021
 > 7.024975024975025
 > 6.904191616766467
 > 6.767232767232767
Average (mean) 6.7064305455024025

fetchGET
Raw:
 > 5.562437562437562
 > 6.1678321678321675
 > 5.814185814185814
 > 6.281718281718281
 > 6.185814185814186
 > 6.275449101796407
 > 6.233766233766234
 > 6.197802197802198
Average (mean) 6.089875693169105

requestGET
Raw:
 > 5.460539460539461
 > 5.616383616383616
 > 5.64035964035964
 > 5.4245754245754245
 > 5.088911088911089
 > 5.430569430569431
 > 4.891108891108891
 > 4.945054945054945
Average (mean) 5.312187812187812

requestPromiseGET
Raw:
 > 4.987012987012987
 > 5.088911088911089
 > 5.178821178821178
 > 5.2298507462686565
 > 4.753246753246753
 > 5.100899100899101
 > 5.238850346878097
 > 5.370629370629371
Average (mean) 5.118527696583404

Winner: roiGET
Compared with next highest (wreckGET), it's:
1.69% faster
1.02 times as fast
0.01 order(s) of magnitude faster
BASICALLY THE SAME

Compared with the slowest (requestPromiseGET), it's:
24.97% faster
1.33 times as fast
0.12 order(s) of magnitude faster
```

second time:

```
```

third time:

```
```

**Comment by helio-frota** [comment]
https://github.com/bucharest-gold/roi/commit/33036685d6aa115b2b57c2d1ec60faba087668ec

**Comment by helio-frota** [comment]
Additional info:
```
1,69 - 0,47 - 5,99  [ our current test ]
4.81 - 4.16 - 4.68 [ with GC working all the time ]
4.92 - 6.02 - 3.56 - 2.83 [ Avoiding GC ]
```
